gemrb/core/GUI/TextArea.cpp

```diff
@@ -648,7 +648,7 @@ void TextArea::SetSelectOptions(const std::vector<SelectOption>& opts, bool numb
 		TextContainer* selOption = new TextContainer(optFrame, ftext, palettes[PALETTE_OPTIONS]);
 		if (numbered) {
 			wchar_t optNum[6];
-			swprintf(optNum, sizeof(optNum)/sizeof(optNum[0]), L"%zu. - ", i+1);
+			swprintf(optNum, static_cast<int32_t>(sizeof(optNum)/sizeof(optNum[0])), L"%d. - ", i+1);
```

Comment:
Codacy found an issue: [%d in format string (no. 1) requires 'int' but the argument type is 'size_t {aka unsigned long}'.](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6347296)